### PR TITLE
Maps

### PR DESCRIPTION
**Pull request by abhinavpappu**
The maps are a bit too small on mobile, but fixing this requires changing Backdrop, which I'll do after the other pages using Backdrop are merged in so that it doesn't break anything

### CODE REVIEW

File: src/components/NavBar/index.jsx [hunk 1]
```diff
@@ -7,11 +7,11 @@ import logoWhite from 'assets/logo_white.svg';
 import menu from 'assets/icons/menu.svg';
 
 const links = [
-  // { text: 'Schedule', to: '/schedule' },
-  // { text: 'Maps', to: '/maps' },
-  // { text: 'Prizes', to: '/prizes' },
-  // { text: 'Travel', to: '/travel' },
-  // { text: 'Projects', to: '/projects' },
+  { text: 'Schedule', to: '/schedule' },
+  { text: 'Maps', to: '/maps' },
```
Comment (brianstrauch) [inline src/components/NavBar/index.jsx:11]: Only uncomment your section

File: src/scenes/Maps/style.scss [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      }
+    }
+
+    .location-select {
+      position: absolute;
+      left: 50px;
+      top: 10px;
+      width: 200px;
+      pointer-events: none;
+      svg { // the dropdown arrow
```
Comment (brianstrauch) [inline src/scenes/Maps/style.scss]: Can you give this an id? Better than a comment

**Review by brianstrauch — CHANGES_REQUESTED** [review]
Looks good. Made two comments, and we still need a background on mobile (it's white). I think Aspen forgot to include the background in his mobile designs

File: src/scenes/Maps/style.scss (hunk 2 above)
Comment (abhinavpappu) [reply src/scenes/Maps/style.scss]: It's inside the react-select component, so I can't give it an id directly

Comment (abhinavpappu) [reply src/scenes/Maps/style.scss]: Never mind

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**